.azure-pipelines/testscripts_analyse/analyse_testscripts.py

```diff
@@ -130,8 +130,16 @@ def get_PRChecker_scripts():
         logging.error('Failed to load file {}, error {}'.format(f, e))
 
     topology_type_pr_test_scripts = {}
+    topology_type_pr_test_skip_scripts = {}
```

Comment:
Nit: Dictionaries can have a naming convention like `topology_type_pr_test_scripts` -> `test_scripts_per_topology_type` and `topology_type_pr_test_skip_scripts` -> `skipped_scripts_per_topology_type`

Reply:
Thank you for suggestion, changed.